### PR TITLE
Make commit with thoughts from #120 description

### PR DESCRIPTION
**Pull request by Ericson2314**
CC @cgibbard

### CODE REVIEW

File: frontend/Rhyolite/Frontend/App.hs [hunk 1]
```diff
@@ -74,7 +74,10 @@ functorToWire
      , QueryResult (q SelectedCount) ~ v SelectedCount)
   => QueryMorphism (q SelectedCount) (q ())
 functorToWire = QueryMorphism
-  { _queryMorphism_mapQuery = mapMaybe (\n -> if n <= 0 then Nothing else Just ())
+  { -- TODO in principle the <= 0 case should never happen.
+    -- Perhaps we should error / impure log / always `Just ()` / etc.,
+    -- but we'd need to track down a bunch of instsances first.
```
Comment (alexfmpe) [inline frontend/Rhyolite/Frontend/App.hs]: *instances

Comment (Ericson2314) [reply frontend/Rhyolite/Frontend/App.hs]: Can you make it a suggestion so I can apply it?

Comment (alexfmpe) [reply frontend/Rhyolite/Frontend/App.hs]: Done

Comment (Ericson2314) [reply frontend/Rhyolite/Frontend/App.hs]: Thanks!

**Comment by ali-abrar** [comment]
This really ought to be an error

**Comment by 3noch** [comment]
@ali-abrar That's what I thought too however I think we may not be ready to break just about every app using this. I want to beef up MonoidMap so we can actually start using it and then pull the trigger there.

**Pull request merged**